src/libs/actions/Device/getDeviceInfo/getOSAndName/index.native.ts

```diff
@@ -1,11 +1,16 @@
 import Str from 'expensify-common/lib/str';
 import RNDeviceInfo from 'react-native-device-info';
+import GetOSAndName from './types';
 
-export default function getOSAndName() {
+const getOSAndName: GetOSAndName = () => {
     const deviceName = RNDeviceInfo.getDeviceNameSync();
     const prettyName = `${Str.UCFirst(RNDeviceInfo.getManufacturerSync() || '')} ${deviceName}`;
     return {
+        // eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
Can we add a comment saying why we need to disable the linter here? I'm guessing it's because we don't chose the parameter names here.